api/Document.json

```diff
@@ -5425,10 +5425,12 @@
               "version_added": "32"
             },
             "safari": {
-              "version_added": null
+              "version_added": "5.1",
+              "alternative_name": "webkitfullscreenchange"
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": "5.1",
```

Comment:
5.1 + 5.1 aren't matching versions of Safari and iOS. Is this based on mirroring or testing?

Also, I think the fullscreen API was supported much later on iOS and may still not be fully supported, so some testing of this on real devices would be great.

Reply:
All the iOS/iPadOS data has been fixed!  5.1 was based upon mirroring, but looking around, there were many other entries with incorrect data, so I fixed them up as well.